integrations/aws/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improvements
 
+- Added _target='blank' attribute to spec links to open a new browser tab instead of the current browser. (0.2.19)
 - Bumped ocean version to ^0.9.7 (#1)
```

Comment:
This should be on it's own as

### Improvements
 - Bumped ocean version to ^0.9.7